test/specs/elements/Flag/Flag-test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import Flag from 'src/elements/Flag/Flag'
+import * as common from 'test/specs/commonTests'
+
+describe('Flag', () => {
+  common.isConformant(Flag)
+
+  it('renders an <i /> element', () => {
+    shallow(<Flag />)
+      .should.have.tagName('i')
+  })
+
+  it('adds the "flag" class', () => {
```

Comment:
This test is covered by the `isConformant()` test.  The `isConformant()` test ensures the className includes has the component name.